### ISSUE TITLE
Regarding Insertion of Reocrd in mongo db

### ISSUE THREAD

**Issue by pra0909** [IOT-DSA/dslink-dart-mongodb-controller#7]
I wan to use mongo DSLink to insert record  in db. But  option is not available (P.S. i have gone through wiki also and know this feature is not available), some how is  it possible or any other dslink by which i can achieve this thing. Any suggestion will be helpful for me.

**Comment by Pacane** [comment]
If I'm not mistaken, the old mongo dslink could do inserts, although it needed your database to have full read/write access.

In an ideal world, you would use this new `dslink-dart-mongodb-controller` link. However we've only had requests for read only operations so far, so the writes are not implemented yet. It should not be too hard to implement though.

**Comment by pra0909** [comment]
yeS, It should not be too hard to implement Insert Document operation. I had not gone through this code base , so can i expect a update on regarding this or should i use another DSLink......

**Comment by Pacane** [comment]
I've started working on it, on this link.

**Comment by pra0909** [comment]
Ok, Thanks. I have seen your successful passed test cases. It's done??

**Comment by Pacane** [comment]
Not yet, I still have things to test/integrate. Like right now I don't think there's any way to add date fields.

I still have some work to do.

**Comment by pra0909** [comment]
how can i insert a document with old DSLink which is dslink-dart-mongodb? When i am trying to insert a document via raw js , it shows me disconnected ; but data is inserted to db?

**Comment by Pacane** [comment]
You might want to check the logs to know what happened. But otherwise, I'm not too familiar with the old link, so I don't really know.

**Comment by mayankshrm77** [comment]
Hi, 
Is Insert feature implemented?
